docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -470,24 +470,70 @@ Collection versions use `Semantic Versioning <https://semver.org/>`_ for version
 
 .. _migrate_to_collection:
 
-Migrating Ansible content to a collection
-=========================================
+Migrating Ansible content to a different collection
+====================================================
+
+To migrate content from one collection to another, you need to create three PRs as follows:
+
+#. Create a PR against the old collection to remove the content.
+#. Create a PR against the new collection to add the files removed in step 1.
+#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+
+
+Removing the content from the old collection
+----------------------------------------------
+
+Create a PR against the old collection repo to remove the modules, module_utils, plugins, and docs_fragments related to this migration:
+
+#. If you are moving an action plugin, include the corresponding module with documentation.
+#. If you are moving a module, move any corresponding action plugin that should stay with it.
```

Comment:
```suggestion
#. If you are moving a module, remove any corresponding action plugin that should stay with it.
```

Reply:
@felixfontein @gundalow same question - I think it should stay as move.

Reply:
I'd argument for `remove` as before. It should definitely be the same word though, no matter which one :)